doc/source/reference/general_functions.rst

```diff
@@ -58,6 +58,7 @@ Top-level dealing with datetimelike data
    period_range
    timedelta_range
    infer_freq
+   guess_datetime_format
```

Comment:
For this entry specifically I think you'll need a new autosummary section with 

`.. currentmodule:: pandas.tseries.frequencies`

Reply:
Tried adding a new autosummary section with 
`.. currentmodule:: pandas.tseries.frequencies`
as follows:
```
Top-level dealing with datetimelike data
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.. autosummary::
   :toctree: api/


   to_datetime
   to_timedelta
   date_range
   bdate_range
   period_range
   timedelta_range
   infer_freq


.. autosummary::
   :currentmodule: pandas.tseries.frequencies


   guess_datetime_format
```

However, I'm still getting hit with the `"AttributeError: module 'pandas' has no attribute 'guess_datetime_format'"` for docstring validation - 

should this be a completely different section with a different header?